lib/rubocop/cop/rails/find_by.rb

```diff
@@ -23,6 +23,7 @@ class FindBy < Cop
 
         def on_send(node)
           return unless where_first?(node)
+          return if where_with_method_as_param?(node)
```

Comment:
Yes, I also wanted to split it to: `where_with_variable_as_param?` and `where_with_method_as_param?`.